src/drivers/magnetometer/ak09916/ak09916.hpp

```diff
@@ -85,30 +56,13 @@
 
 #define AK09916REG_WIA           0x00
 
-#define AK09916REG_HXL        0x11
-#define AK09916REG_HXH        0x12
-#define AK09916REG_HYL        0x13
-#define AK09916REG_HYH        0x14
-#define AK09916REG_HZL        0x15
-#define AK09916REG_HZH        0x16
 #define AK09916REG_ST1        0x10
-#define AK09916REG_ST2        0x18
 #define AK09916REG_CNTL2          0x31
 #define AK09916REG_CNTL3          0x32
 
-
-#define AK09916_CNTL2_POWERDOWN_MODE            0x00
-#define AK09916_RESET							0x01
+#define AK09916_RESET				0x01
```

Comment:
So in your opinion we should only have defines for registers we're using? I guess that makes sense because all others are not tested and might potentially be wrong, so you need to look at the datasheet anyway for a change.

Reply:
Yes, exactly. It's also yet another source of perceived complexity. With all these deletions you can see just how little of these drivers we actually use.